homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -130,12 +130,14 @@ def unique_id(self):
     @property
     def device_info(self):
         """Return the device info."""
-        return {
-            "identifiers": {(DOMAIN, self.serial)},
-            "name": self.name,
-            "manufacturer": "Lutron",
-            "model": self._device["model"],
-        }
+        if "serial" in self._device:
```

Comment:
Would it be more clean to overwrite `device_info` in the binary sensor occupancy entity?

Reply:
I guess it is. It's consistent with the original author's intention, which is overwrite properties in sensor entity because it's special.

I'll make the change and run it with jsb5151 for a quick test later today.